lib/pronto/spell.rb

```diff
@@ -26,17 +39,51 @@ def new_message(word, line)
       level = :warning
 
       suggestions = speller.suggestions(word)
-      msg = "#{word} might not be spelled correctly. Spelling suggestions: #{suggestions}"
+
+      msg = %("#{word}" might not be spelled correctly.)
+      unless suggestions.empty?
+        msg += " Spelling suggestions: #{suggestions[0..2].join(', ')}"
+      end
 
       Message.new(path, line, level, msg, nil, self.class)
     end
 
     def speller
-      @speller ||= begin
-        result = FFI::Aspell::Speller.new('en_US')
-        result.suggestion_mode = 'fast'
-        result
+      @speller ||= FFI::Aspell::Speller.new(
+        'en_US', :'sug-mode' => suggestion_mode
+      )
+    end
+
+    def spelling_config
+      @spelling_config ||= begin
+        config_path = File.join(repo_path, CONFIG_FILE)
+        File.exist?(config_path) ? YAML.load_file(config_path) : {}
       end
     end
+
+    def suggestion_mode
+      spelling_config['suggestion_mode'] || 'fast'
+    end
+
+    def min_word_length
+      spelling_config['min_word_length'] || 5
+    end
+
+    def max_word_length
+      spelling_config['max_word_length'] || Float::INFINITY
+    end
+
+    def max_suggestions_number
```

Comment:
@povilasjurcys This method is never used.

Reply:
@zhuravel good point! I'll make a PR to fix it.

Reply:
@zhuravel @povilasjurcys please take a look: https://github.com/prontolabs/pronto-spell/pull/4 🙂